extra/completions/tmatrix.bash

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
Completion script doesn't need this but it will probably need a licensing header.